### PR TITLE
tests for: drop-downs on new bug report page

### PR DESCRIPTION
**Pull request by annajsoleynik**

### CODE REVIEW

File: test/new-bug-report-page.js [hunk 1]
```diff
@@ -198,6 +198,112 @@ describe ('Drop-down lists', function () {
...
+            assert.equal(fontSize.value, data.newBugReportPage.fontSize);
+        });
+
+
+        it(`Font weight should be ${data.newBugReportPage.fontWeight}`, function () {
+            let fontWeight = browser.getCssProperty(locator.newBugReportPage.priorities, 'font-weight');
+            assert.equal(fontWeight.value, data.newBugReportPage.fontWeight);
+        });
+
+        // it(`Font color should be ${data.newBugReportPage.fontColor}`, function () {
```
Comment (artsenius) [inline test/new-bug-report-page.js]: What are those commented tests?

File: test/new-bug-report-page.js [hunk 2]
```diff
@@ -198,6 +198,112 @@ describe ('Drop-down lists', function () {
...
+        //     let fontColor = browser.getCssProperty(locator.newBugReportPage.priorities, 'color');
+        //     assert.equal(fontColor.value, data.newBugReportPage.fontColor);
+        // });
+
+        it(`Text align should be ${data.newBugReportPage.textAlignLeft}`, function () {
+            let textAlign = browser.getCssProperty(locator.newBugReportPage.priorities, 'text-align');
+            assert.equal(textAlign.value, data.newBugReportPage.textAlignLeft);
+        });
+        // it('Verify that the list stretches to the full width of the page', function () {
```
Comment (artsenius) [inline test/new-bug-report-page.js]: this

File: test/new-bug-report-page.js [hunk 3]
```diff
@@ -198,6 +198,112 @@ describe ('Drop-down lists', function () {
...
+            assert.equal(fontSize.value, data.newBugReportPage.fontSize);
+        });
+
+
+        it(`Font weight should be ${data.newBugReportPage.fontWeight}`, function () {
+            let fontWeight = browser.getCssProperty(locator.newBugReportPage.severities, 'font-weight');
+            assert.equal(fontWeight.value, data.newBugReportPage.fontWeight);
+        });
+
+        // it(`Font color should be ${data.newBugReportPage.fontColor}`, function () {
```
Comment (artsenius) [inline test/new-bug-report-page.js]: this

File: test/new-bug-report-page.js [hunk 4]
```diff
@@ -198,6 +198,112 @@ describe ('Drop-down lists', function () {
...
+        //     let fontColor = browser.getCssProperty(locator.newBugReportPage.severities, 'color');
+        //     assert.equal(fontColor.value, data.newBugReportPage.fontColor);
+        // });
+
+        it(`Text align should be ${data.newBugReportPage.textAlignLeft}`, function () {
+            let textAlign = browser.getCssProperty(locator.newBugReportPage.severities, 'text-align');
+            assert.equal(textAlign.value, data.newBugReportPage.textAlignLeft);
+        });
+        // it('Verify that the list stretches to the full width of the page', function () {
```
Comment (artsenius) [inline test/new-bug-report-page.js]: this

File: test/new-bug-report-page.js [hunk 5]
```diff
@@ -198,6 +198,112 @@ describe ('Drop-down lists', function () {
...
+            assert.equal(fontSize.value, data.newBugReportPage.fontSize);
+        });
+
+
+        it(`Font weight should be ${data.newBugReportPage.fontWeight}`, function () {
+            let fontWeight = browser.getCssProperty(locator.newBugReportPage.priorities, 'font-weight');
+            assert.equal(fontWeight.value, data.newBugReportPage.fontWeight);
+        });
+
+        // it(`Font color should be ${data.newBugReportPage.fontColor}`, function () {
```
Comment (artsenius) [inline test/new-bug-report-page.js]: this

File: test/new-bug-report-page.js [hunk 6]
```diff
@@ -198,6 +198,112 @@ describe ('Drop-down lists', function () {
...
+        //     let fontColor = browser.getCssProperty(locator.newBugReportPage.priorities, 'color');
+        //     assert.equal(fontColor.value, data.newBugReportPage.fontColor);
+        // });
+
+        it(`Text align should be ${data.newBugReportPage.textAlignLeft}`, function () {
+            let textAlign = browser.getCssProperty(locator.newBugReportPage.priorities, 'text-align');
+            assert.equal(textAlign.value, data.newBugReportPage.textAlignLeft);
+        });
+        // it('Verify that the list stretches to the full width of the page', function () {
```
Comment (artsenius) [inline test/new-bug-report-page.js]: this

**Comment by annajsoleynik** [comment]
these tests are failed, text on that fields is not #495057 as it says in
test cases. I filed bug.

**Comment by annajsoleynik** [comment]
and i am still working on that ones with WIDTH

File: test/new-bug-report-page.js (hunk 6 above)
Comment (artsenius) [reply test/new-bug-report-page.js]: Fixed the bug. Please, check. And also, finish test for Width.

**Pull request merged**